nixos/modules/services/security/fail2ban.nix

```diff
@@ -118,68 +118,68 @@ in
         default = false;
         type = types.bool;
         description = lib.mdDoc ''
-          Allows to use database for searching of previously banned ip's to increase
-          a default ban time using special formula, default it is banTime * 1, 2, 4, 8, 16, 32...
+          "bantime.increment" allows to use database for searching of previously banned ip's to increase
+          a default ban time using special formula, default it is banTime * 1, 2, 4, 8, 16, 32 ...
         '';
       };
 
       bantime-increment.rndtime = mkOption {
```

Comment:
We should adjust the option names, too, shouldn't we?

Reply:
https://github.com/NixOS/nixpkgs/pull/226693#discussion_r1171031885